domain/tutoringModel/utils.py

```diff
@@ -113,3 +113,33 @@ def random_generator(num, size, type_):
         return rng.choice([True, False], size)
     else:
         raise err.WrongParameterValueError()
+
+
+# Check if learning style is correctly formatted and
+# has the right dimensions
+def check_learning_style(input_learning_style):
+    condition1 = "perception_dimension" in input_learning_style.keys()
```

Comment:
hardcoded strings best idea? Maybe constant somehwere in constants or utils. Since this gets used in different parts of the application.

Reply:
Changed it here, maybe need a new issue for changing it in the whole application